game/scripts/vscripts/Sections/Chapter4/SectionOpening.ts

```diff
@@ -33,6 +33,7 @@ const firstScanLocation = Vector(2000, 3800);
 const secondScanLocation = Vector(-2000, 3800);
 const outpostHighgroundCenter = Vector(-1800, 4000);
 let currentRequiredScanLocation = firstScanLocation;
+let waitingForPlayerToScan = false
```

Comment:
This needs to be set to false in onStart. Right now its in the global scope, so on the second run it could still be true.

Reply:
You're right, I'll fix it after work today.

Reply:
Fixed! Please merge.